koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -687,11 +712,15 @@ def handler(self, src, target, opts=None):
             self.check_whitelist(data[LABEL_DATA_MAP['COMPONENT']], target_info)
 
         try:
-            auto_release = (data[LABEL_DATA_MAP['RELEASE']] ==
-                            LABEL_DEFAULT_VALUES['RELEASE'])
-            if auto_release:
-                # Do not expose default release value
-                del data[LABEL_DATA_MAP['RELEASE']]
+            # Flatpak builds append .<N> to the release generated from module version
+            if flatpak:
+                auto_release = True
```

Comment:
This skips the `del` a few lines down. Is that a problem?

Reply:
For the flatpak case the release field of data, if present, is not a meaningless object(), it's the release that we'll be appending .<N> onto. I suppose it could be considered confusing that it's not literally the release of the resulting artifact, but I think it's ifne.